### DISCUSSION TITLE
Rename DCC guides

### DISCUSSION THREAD

**Discussion by edvinskis** [TU-Delft-DCC/TU-Delft-DCC.github.io#131] [Polls]
Thank you @manuGil for bringing this to our attention in https://github.com/TU-Delft-DCC/TU-Delft-DCC.github.io/issues/98#issuecomment-2230145464.

The guides are currently accessible at https://tu-delft-dcc.github.io/. To ensure we choose a name that resonates with our community, I'd like to open a poll.

Please vote for your preferred option or leave a comment with a new suggestion. Your input is valuable in helping us select a name that best represents our guides and their purpose.

@TU-Delft-DCC/tu-delft-dcc

**Comment by joaoxg** [comment]
tudelft-dcc-guides.github.io

**Comment by mwakok** [comment]
Perhaps we should create the domain(s) guides.dcc.tudelft.nl and/or tudelft.nl/dcc/guides

**Reply by joaoxg** [reply]
I like guides.dcc.tudelft.nl or dcc.tudelft.nl/guides. I think that "dcc.tudelft.nl" should always be there, and whichever way we choose to append the guides should be done with other initiatives like OOH (we are also working on a short link for the application page)

**Reply by joaoxg** [reply]
Hmm I was not aware of the tudelft.nl/dcc short link. If we decide to promote ourselves with the tudelft.nl/dcc shortlink, then I would prefer tudelft.nl/dcc/guides.

Is this something already decided,  or should we discuss this tomorrow?

**Reply by manuGil** [reply]
I think, we shouldn't overthink this too much. Choose a name for the repo, the shorter and meaningful,  the better.  The TU Delft urls are something different
